pkgs/tools/misc/diskonaut/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "diskonaut";
+  version = "1.15.1";
```

```suggestion
  version = "0.3.0";
```

This version looks wrong? The latest upstream is 0.3.0:

https://github.com/imsnif/diskonaut/releases